Project.toml

```diff
@@ -1,31 +1,30 @@
 name = "ReferenceTests"
 uuid = "324d217c-45ce-50fc-942e-d289b448e8cf"
 authors = ["Christof Stocker <stocker.christof@gmail.com>", "Lyndon White <oxinabox@ucc.asn.au>"]
-version = "0.7.0"
+version = "0.8.0"
 
 [deps]
-ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 DeepDiffs = "ab62b9b5-e342-54a8-a765-a90f495de1a6"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 ImageInTerminal = "d8c32880-2388-543b-8c61-d9f865259254"
-Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-ColorTypes = ">= 0.4"
 FileIO = ">= 0.4"
+ImageCore = ">= 0.8.1"
 ImageInTerminal = ">= 0.3"
-Images = ">= 0.15"
-julia = ">= 1.0"
+julia = "1"
```

Comment:
Can we change all of theses to use SemVer bounds?
I ie remove the `>=1` ?

Reply:
Agreed since `1.2.3` == `^1.2.3` <--> `[1.2.3, 2.0.0)`

Reply:
Did I understand it mistakenly here since it's restricted to ImageInTerminal `[0.3.0, 0.4.0)` (not as the documentation says)

Reply:
Oh I guess `0.y.z` is treated differently?